proto/cheqd/v1/query.proto

```diff
@@ -13,6 +13,9 @@ service Query {
 	rpc Did(QueryGetDidRequest) returns (QueryGetDidResponse) {
 		option (google.api.http).get = "/cheqd/cheqdnode/cheqd/did/{id}";
 	}
+	rpc DidJson(QueryGetDidJsonRequest) returns (QueryGetDidJsonResponse) {
+		option (google.api.http).get = "/cheqd/cheqdnode/cheqd/did+json/{id}";
```

Comment:
Why is the URL path to fetch this so weird? Added a comment over on ClickUp for extended discussion: https://app.clickup.com/t/6600954/DEV-957?comment=560490023